### ISSUE TITLE
[1.15.2] Dungeons spawning way too many mobs with identical(ish) gears

### ISSUE THREAD

**Issue by Dallong** [xyroc/DungeonCrawl#46]
Hi, first submitting issue here, and english isn't my first language so sorry in advance if my words do not make sense, and thank you for your patience.
I'm hosting a modded server and using dungeon crawl mod version 2.1.1, playing with few friends. And we have some spawning issue. Maybe it's something to do with compatibility, as I am using apotheosis. Which is make changes to spawner.(but it was never an issue in 1.14.4 server, and 1.12.2 single player world.)
To summarize, my friend once found dungeon's lower chamber, that built with nether stuff. She broke into it, and as she got near the spawner, it spawn way too many mobs. Wearing identical gears. 
And it was not happened once, as me and my friend was doing some caving together, and we come across with a dungeon. And... Well, same thing happens.

![1603072235033](https://user-images.githubusercontent.com/61864609/96393408-f6cb9c80-11f9-11eb-852e-f0d6214c5aa6.png)

My friend took a screenshot. As you can see, too many mobs (and trust me, we have been through few dungeons on other version of this mod to not see that as normal.) with identical(ish) items. I say ish because there was mobs with different gears, but it doesn't feel normal to any of us. 

Any help or advice will helpful, and if you need any more information, I will try my best to provide them. Thanks!

**Comment by Unhook** [comment]
Can confirm in latest 1.16.3 version as well.

**Comment by Dallong** [comment]
Good to see someone commented. :D
As for spawners.. well, we ended up setting farm using that. Lots and lots of gears to use, or just salvage using other mod's salvager.

Having said that, I've come across with patterns. On one spawner that we've kept farming, there was few groups that spawned. At light level 0, spawner for wither skeleton spawned; group of five or six wither skeleton that had a golden leggings, and boot that all has same enchantments. Other group was wearing iron armor, equipping golden shovel also has same enchantments. 
Testing with other spawners right now. And having same result.
